2 Niagara Fundamentals Exercise - Multiplexer/multiplexer-rt/srcTest/com/tridiumuniversity/multiplexer/BSuperMultiplexerTest.java

```diff
@@ -0,0 +1,119 @@
+package com.tridiumuniversity.multiplexer;
+
+import org.testng.Assert;
+import org.testng.annotations.*;
+
+import javax.baja.nre.annotations.NiagaraType;
+import javax.baja.sys.*;
+import javax.baja.test.BTest;
+import javax.baja.test.BTestNg;
+
+@NiagaraType
+public class BSuperMultiplexerTest extends BTestNg
+{
+//region /*+ ------------ BEGIN BAJA AUTO GENERATED CODE ------------ +*/
+//@formatter:off
+/*@ $com.tridiumuniversity.multiplexer.BSuperMultiplexerTest(2979906276)1.0$ @*/
+/* Generated Tue Feb 20 11:33:02 CST 2024 by Slot-o-Matic (c) Tridium, Inc. 2012-2024 */
+
+  //region Type
+
+  @Override
+  public Type getType() { return TYPE; }
+  public static final Type TYPE = Sys.loadType(BSuperMultiplexerTest.class);
+
+  //endregion Type
+
+//@formatter:on
+//endregion /*+ ------------ END BAJA AUTO GENERATED CODE -------------- +*/
+
+	private TestStationHandler handler;
+	private BStation station;
+	private BSuperMultiplexer multiplexer;
+
+
+	@BeforeClass(alwaysRun=true)
+	public void setup() throws Exception {
+		handler = BTest.createTestStation();
```

Comment:
You can just use `createTestStation()` since this class subclasses `BTestNg`, which subclasses `BTest`.